bin/compile

```diff
@@ -440,7 +445,14 @@ case "${TOOL}" in
     gomodules)
         pkgs=${GO_INSTALL_PACKAGE_SPEC:-$(awk '{ if ($1 == "//" && $2 == "+heroku" && $3 == "install" ) { print substr($0, index($0,$4)); exit }}' ${goMOD})}
         if [ -z "${pkgs}" ]; then
-            pkgs="default"
+            if [ -d "${build}/cmd" ]; then
+                step "Detected \`cmd\` folder. Installing packages under \`./cmd\`"
+                step "To install packages under MyDir/ add the following to \`go.mod\` file:"
+                warn "// +heroku install ./MyDir/..."
+                pkgs="./cmd/..."
```

Comment:
I would suggest adding a comment with a link to https://github.com/golang-standards/project-layout#go-directories

Reply:
I will add this: Supports standard go directory structure with the link